synfig-studio/plugins/lottie-exporter/properties/shapePropKeyframe/helper.py

```diff
@@ -275,6 +275,6 @@ def quadratic_to_cubic(qp0, qp1, qp2):
     """
     cp0 = qp0
     cp3 = qp2
-    cp1 = qp0 + 2/3*(qp1 - qp0)
-    cp2 = qp2 + 2/3*(qp1 - qp2)
+    cp1 = qp0 + 2/float(3)*(qp1 - qp0)
```

Comment:
I think you can just use 3.0 here

Reply:
Made the changes which you've suggested.